pyproject.toml

```diff
@@ -38,6 +38,7 @@ allow_no_feature_nodes = [
     "tests/parametric/test_headers_precedence.py",
     "tests/parametric/test_headers_tracecontext.py",
     "tests/parametric/test_headers_tracestate_dd.py",
+    "tests/parametric/test_otel_interoperability.py",
```

Comment:
Could you create the feature in the feature parity dashboard, and link those test to it. Please ping me if you need any help.

Reply:
I added [this feature](https://dd-feature-parity.azurewebsites.net/Manage/Features/Edit?id=285) and linked it to the path of this test (although not yet merged). Is that OK? I tried to follow this [documentation](https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2919924286/Feature+Parity) 😃

Reply:
You can update the manual status (even if it's not required, it's always good to have this info). But this status will be validated with test results (eg shown if and only if test are ok).

Though, you need to add this feature in `utils/_features.py` (big boilerplate file), and decorate your class/method with the good `@feature`, and then, remove the added line in pyproject.

Then all good !